cmd/upgrade/roles/cmd.go

```diff
@@ -109,9 +109,8 @@ func run(cmd *cobra.Command, argv []string) error {
 		aws.SetModeKey(argv[0])
 		ocm.SetClusterKey(argv[1])
 		skipInteractive = true
-		if len(argv) > 2 && argv[2] != "" {
-			args.clusterUpgradeVersion = argv[2]
-		}
+		args.clusterUpgradeVersion = argv[2]
```

Comment:
Coming from cluster upgrade it should always have all the argvs, checking didn't make sense